portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormRules.testcase

```diff
@@ -1087,4 +1087,69 @@ definition {
 			locator1 = "FormRules#RULE_BUILDER_FIELD_SELECT_VALUE");
 	}
 
+	@description = "LRQA-47081 - Verify that rules are working for multiple selection field"
+	@priority = "4"
+	test VerifyTheRulesForMultipleSelection {
```

Comment:
@WYZ-Issac, we can use the name `CanUseRuleOnMultipleSelectionField` - we want the test name to tell other test engineers whether a test can/cannot do something.  While something like "UseRuleOnMultipleSelectionField" or "AddRuleOnMultipleSelectionField" may also work, neither really says what we should expect.